source/tyr/tyr/binarisation.py

```diff
@@ -1047,3 +1066,69 @@ def poi2mimir(self, instance_name, input, autocomplete_version, job_id=None, dat
             models.db.session.commit()
 
         raise
+
+
+@celery.task(bind=True)
+def fusio2s3(self, instance_config, filename, job_id, dataset_uid):
+    """ Zip fusio file and launch fusio2s3 """
+
+    job = models.Job.query.get(job_id)
+    dataset = _retrieve_dataset_and_set_state("fusio", job.id)
+    instance = job.instance
+
+    logger = get_instance_logger(instance, task_id=job_id)
+    try:
+        filename = zip_if_needed(filename)
```

Comment:
I'm not sure if the zip is necessary

Reply:
If we receive a zip we do nothing, it's just a security in the case we receive a folder of fusio/gtfs data